packages/account/src/configs.ts

```diff
@@ -1,4 +1,4 @@
 export const DEVNET_NETWORK_URL = 'https://devnet.fuel.network/v1/graphql';
 export const TESTNET_NETWORK_URL = 'https://testnet.fuel.network/v1/graphql';
-// TODO: replace placeholder with mainnet network url
-// export const NETWORK_URL = '';
+
+export const MAINNET_NETWORK_URL = 'https://mainnet.fuel.network/v1/graphql';
```

Comment:
We don't want to provide constants with URLs for users anymore because we may soon have several other URLs from different providers, and we don't want to concentrate everything on a single point of failure.

Yes, users may still be using the existing constants for Devnet/Testnet, but _deprecating_ is safe and will give them time to adapt before we remove them altogether—which we should do when we release the next breaking changes.

That's why I modified the issue to:
 1. Do not add another constant
 2. Deprecate the existing ones

Reply:
Got it. I interpreted the issue as we still wanted the new constant due to the todo comment in our code. I made the corrections